### PR TITLE
NO-ISSUE: Update bug-master configurations

### PR DESCRIPTION
**Pull request by danmanor**

### CODE REVIEW

**Comment by danmanor** [comment]
@gamli75 Can you please comment what other changes should be made ?

File: config/channel/assisted-deployment-ci/bug_master_configuration.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ assignees:
     - job_name: e2e-nutanix-assisted
       startswith: true
       users:
-        - dmanor
+        - agentil
```
Comment (gamli75) [inline config/channel/assisted-deployment-ci/bug_master_configuration.yaml]: please change it to Eran Ifrach

**Review by gamli75 — COMMENTED** [review]
please move e2e-ai-operator-ztp-hypershift-zero-nodes-periodic to Crystal and change all the 411 jobs to newer ones, for example: e2e-ai-operator-ztp-ipv4v6-3masters-ocp-411-periodic

**Comment by danmanor** [comment]
@gamli75  Should I assign all this bundle to Crystal ?
```
        - e2e-ai-operator-ztp-disconnected-periodic
        - e2e-metal-assisted-day2-periodic
        - e2e-ai-operator-ztp-hypershift-zero-nodes-periodic
```

Or just the last one ?

**Comment by gamli75** [comment]
> @gamli75 Should I assign all this bundle to Crystal ?
> 
> ```
>         - e2e-ai-operator-ztp-disconnected-periodic
>         - e2e-metal-assisted-day2-periodic
>         - e2e-ai-operator-ztp-hypershift-zero-nodes-periodic
> ```
> 
> Or just the last one ?

just the last one

**Pull request merged**